crates/editor/src/mouse_context_menu.rs

```diff
@@ -174,6 +173,7 @@ pub fn deploy_context_menu(
                         deployed_from_indicator: None,
                     }),
                 )
+                .action("Reformat code", Box::new(Format))
```

Comment:
We should call this `Format Buffer`.

```suggestion
                .action("Format Buffer", Box::new(Format))
```

Reply:
Should it format the whole file or a selection(multiselection maybe?) as well?

Reply:
I don't think we support formatting in selections right now, so it would just dispatch the `Format` action and format the entire buffer.

Reply:
There's a prototype for the formatting selections support higher in the comments. But I agree that for now we can just add a Format Buffer action